jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/ColibriV2SessionManager.kt

```diff
@@ -63,17 +70,21 @@ class ColibriV2SessionManager(
     internal val rtcStatsEnabled: Boolean,
     private val bridgeVersion: String?,
     parentLogger: Logger
-) : ColibriSessionManager {
+) : ColibriSessionManager, Cascade<Colibri2Session, Colibri2Session.Relay> {
     private val logger = createChildLogger(parentLogger)
 
     private val eventEmitter = AsyncEventEmitter<ColibriSessionManager.Listener>(TaskPools.ioPool)
     override fun addListener(listener: ColibriSessionManager.Listener) = eventEmitter.addHandler(listener)
     override fun removeListener(listener: ColibriSessionManager.Listener) = eventEmitter.removeHandler(listener)
 
+    private val topologySelectionStrategy = BridgeConfig.config.topologyStrategy.also {
+        logger.info("Using ${it.javaClass.name}")
+    }
+
     /**
      * The colibri2 sessions that are currently active, mapped by the [Bridge] that they use.
      */
-    private val sessions = mutableMapOf<Bridge, Colibri2Session>()
+    override val sessions = mutableMapOf<String?, Colibri2Session>()
```

Comment:
After this is merged, I volunteer to switch to using the JID instead of the advertised relay ID. Then it won't be nullable.